src/errors/responseError.ts

```diff
@@ -0,0 +1,9 @@
+import { Response } from "express"
+
+export const sendErrorResponse = (
+  res: Response,
+  statusCode: number,
+  message: string
+) => {
+  return res.status(statusCode).json({ status: "ERROR", message })
```

Comment:
for the status message in the json, maybe we can use something like [http-status-codes](https://www.npmjs.com/package/http-status-codes) reason phrase to automatically set the message based on `statusCode`